crm_claim_rma/wizard/account_invoice_refund.py

```diff
@@ -20,27 +21,32 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
-from openerp.osv import orm
 
+from openerp.models import api, TransientModel
+from openerp.fields import Char
 
-class account_invoice_refund(orm.TransientModel):
 
+class AccountInvoiceRefund(TransientModel):
     _inherit = "account.invoice.refund"
 
-    def compute_refund(self, cr, uid, ids, mode='refund', context=None):
+    def _get_description(self):
+        context = self.env.context
         if context is None:
             context = {}
-        if context.get('invoice_ids'):
-            context['active_ids'] = context.get('invoice_ids')
-        return super(account_invoice_refund, self).compute_refund(
-            cr, uid, ids, mode=mode, context=context)
 
-    def _get_description(self, cr, uid, context=None):
-        if context is None:
-            context = {}
         description = context.get('description') or ''
         return description
 
-    _defaults = {
-        'description': _get_description,
-    }
+    description = Char(default=_get_description)
```

Comment:
Can you use fields.Char, I kon it save space but at the end migrations are more difficult.